specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+            "$ref": "#/definitions/GitHubRepo"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubRepoProperties": {
+      "description": "GitHub Repo properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "accountId": {
+          "format": "int64",
+          "description": "Gets or sets gitHub repo account id.",
+          "type": "integer"
+        },
+        "repoUrl": {
+          "description": "Gets or sets gitHub repo url.",
+          "type": "string"
+        },
+        "ownerName": {
+          "description": "Gets or sets GitHub Owner Name.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubReposProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "accountId": {
+          "format": "int64",
+          "description": "Gets or sets gitHub repo account id.",
+          "type": "integer"
+        },
+        "repoName": {
+          "description": "Gets or sets gitHub repo name.",
+          "type": "string"
+        },
+        "repoUrl": {
+          "description": "Gets or sets gitHub repo url.",
+          "type": "string"
+        }
+      }
+    },
+    "ProvisioningState": {
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true,
+        "modelAsExtensible": true
+      }
+    },
+    "RuleCategory": {
+      "enum": [
+        "Code",
+        "Artifacts",
+        "Dependencies",
+        "Secrets",
+        "IaC",
+        "Containers"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "RuleCategory",
+        "modelAsString": true,
+        "modelAsExtensible": true
+      }
+    },
+    "TargetBranchConfiguration": {
+      "description": "Branch onboarding info.",
+      "type": "object",
+      "properties": {
+        "names": {
+          "description": "Gets or sets branches that should have annotations.\r\n\r\nFor Ignite, we will be supporting a single default branch configuration in the UX.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    }
+  },
+  "parameters": {
+    "AzureDevOpsConnectorName": {
+      "in": "path",
+      "name": "azureDevOpsConnectorName",
+      "description": "Name of the AzureDevOps Connector.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "AzureDevOpsOrgName": {
+      "in": "path",
+      "name": "azureDevOpsOrgName",
```

Comment:
>azureDevOpsOrgName [](http://example.com/codeflow?start=15&length=18)

on the naming theme, I'd suggest to call this just 'organizationName'

Reply:
will tackle in next api version.